osgp/platform/osgp-domain-core/src/main/java/org/opensmartgridplatform/domain/core/valueobjects/smartmetering/GetOutagesRequestData.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2015 Smart Society Services B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.domain.core.valueobjects.smartmetering;
+
+import java.io.Serializable;
+
+import org.joda.time.DateTime;
+import org.opensmartgridplatform.domain.core.valueobjects.DeviceFunction;
+import org.opensmartgridplatform.shared.exceptionhandling.ComponentType;
+import org.opensmartgridplatform.shared.exceptionhandling.FunctionalException;
+import org.opensmartgridplatform.shared.exceptionhandling.FunctionalExceptionType;
+
+public class GetOutagesRequestData implements Serializable, ActionRequest {
+
+    /*
+     * (non-Javadoc)
+     * 
+     * @see
+     * org.opensmartgridplatform.domain.core.valueobjects.smartmetering.ActionValueObject
+     * #validate()
+     */
```

Comment:
For me, comments like this look like they haven't got something useful to offer over not adding anything.
You could consider leaving it out.